lib/deployinator/templates/log_table.mustache

```diff
@@ -3,7 +3,7 @@
         <tr>
             <th>
                 {{# prev_page }}
-                <form action="/log">
```

Comment:
what was the point of this change? (or what was the point of this in the first place?)